src/DoctrineWriter.php

```diff
@@ -2,11 +2,11 @@
 
 namespace Port\Doctrine;
 
+use Port\Doctrine\Exception\UnsupportedDatabaseTypeException;
```

Comment:
I wonder why that is. Since you are in the same base namespace `throw new Exception\UnsupportedDatabaseTypeException` should just work. Can you check that?